tests/models/bert/test_tokenization_bert_tf.py

```diff
@@ -4,15 +4,15 @@
 
 from transformers import AutoConfig, TFAutoModel, is_tensorflow_text_available, is_tf_available
 from transformers.models.bert.tokenization_bert import BertTokenizer
-from transformers.testing_utils import require_tensorflow_text, slow
+from transformers.testing_utils import require_tensorflow_text, require_tf, slow
 
 
-if is_tensorflow_text_available():
-    from transformers.models.bert import TFBertTokenizer
-
 if is_tf_available():
     import tensorflow as tf
 
+    if is_tensorflow_text_available():
+        from transformers.models.bert import TFBertTokenizer
```

Comment:
This import will trigger
```
import tensorflow as tf
```
in `src/transformers/models/bert/tokenization_bert_tf.py`